manifests/p/PointPlanck/FileBot/4.9.4/PointPlanck.FileBot.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+PackageIdentifier: PointPlanck.FileBot
+PackageVersion: 4.9.4
+PackageLocale: en-US
+Publisher: PointPlanck
```

Comment:
![image](https://user-images.githubusercontent.com/52826317/128828972-42635942-cf78-4132-ba73-6529feb64b33.png)

```suggestion
Publisher: Point Planck Limited
```

Reply:
Sure. That works too.